src/signals/settings/categories/components/StandardTextsField/StandardTextsField.tsx

```diff
@@ -64,12 +65,17 @@ export const StandardTextsField = ({ name, onChange }: Props) => {
         (status) => status.value === selectedStatus
       )?.key
 
-      const orderedStandardTexts = data.results.filter(
+      let orderedStandardTexts = data.results.filter(
```

Comment:
shorter version: 

const orderedStandardTexts = data.results
  .filter(result => result.state === state)
  .filter(item => !statusOption || item.active.toString() === statusOption)

Reply:
changed it.